src/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.Rsa.cs

```diff
@@ -47,13 +73,29 @@ internal static partial class Crypto
         [DllImport(Libraries.CryptoNative, EntryPoint = "CryptoNative_RsaGenerateKeyEx")]
         internal static extern int RsaGenerateKeyEx(SafeRsaHandle rsa, int bits, SafeBignumHandle e);
 
+        internal static unsafe bool RsaSign(int type, ReadOnlySpan<byte> m, int m_len, Span<byte> sigret, out int siglen, SafeRsaHandle rsa)
+        {
+            fixed (byte* mPtr = &m.DangerousGetPinnableReference(), sigretPtr = &sigret.DangerousGetPinnableReference())
```

Comment:
FWIW I strongly prefer two stacked fixed statements to the `,` declaration syntax.